.dockerignore

```diff
@@ -0,0 +1,4 @@
+.env
+.venv
+.idea
+.vscode
```

Comment:
Minor since you only copy the `todo_app` folder, but you might also want to add e,.g. the caching folders that you might see appear in there:
```
**/__pycache__
```